scripts/plotting/global_latlon_map.py

```diff
@@ -442,6 +375,21 @@ def global_latlon_map(adfobj):
                                 elif (redo_plot) and plot_name.is_file():
                                     plot_name.unlink()
 
+                                #If requested, then calculate the monthly-weighted seasonal averages:
+                                if weight_season:
+                                    mseasons[s] = pf.seasonal_mean(mdata, season=s, is_climo=True)
+                                    oseasons[s] = pf.seasonal_mean(mdata, season=s, is_climo=True)
+                                else:
+                                    #Just average months as-is:
+                                    mseasons[s] = mdata.sel(time=seasons[s], lev=pres).mean(dim='time')
+                                    oseasons[s] = odata.sel(time=seasons[s], lev=pres).mean(dim='time')
+                                    # difference: each entry should be (lat, lon)
+                                    dseasons[s] = mseasons[s] - oseasons[s]
```

Comment:
Should `dseasons` be outside the `else`  block?

Reply:
Yes.